generators/typegen/generator.go

```diff
@@ -126,15 +158,57 @@ func (g *Generator) generateTypesForPackage(s *schema.Schema, genConfig *config.
 			}
 
 			enumsForGen = append(enumsForGen, xxx)
+		case schema.KindScalar:
+			log.Warnf("scalar type: %+v", t)
+
+			// Default scalars to int
+			createAs := "int"
+			skipTypeCreate := false
+
+			for _, p := range pkgConfig.Types {
+				if p.Name == t.GetName() {
+					if p.CreateAs != "" {
+						createAs = p.CreateAs
+					}
+
+					if p.SkipTypeCreate {
+						skipTypeCreate = true
+					}
+				}
+			}
+
+			if !t.IsGoType() && !skipTypeCreate {
+				xxx := goScalar{
+					Description: t.GetDescription(),
+					Name:        t.GetName(),
+					Type:        createAs,
+				}
+
+				scalarsForGen = append(scalarsForGen, xxx)
+			}
 		default:
-			log.Debugf("default reached for kind %s, ignoring", t.Name)
+			log.Debugf("default reached for kind %s, ignoring: %s", t.Name, t.Kind)
```

Comment:
I think the arguments here are backwards.  "Unknown kind `t.Kind`, ignoring `t.Name`"?

Reply:
Oh yeah, they are.